visual_behavior/data_access/loading.py

```diff
@@ -143,7 +143,7 @@ def get_released_ophys_experiment_table(exclude_ai94=True):
     Returns:
         experiment_table -- returns a dataframe with ophys_experiment_id as the index and metadata as columns.
     '''
-    data_storage_directory = '/allen/programs/braintv/workgroups/nc-ophys/visual_behavior/production_cache'
+    data_storage_directory = '//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/production_cache'
```

Comment:
Sorry, if my comment was confusing. I didn't mean take previous path out. I meant write an if/else statement to check for OS and to use one format ("//") if the function is run on Windows, and the other format ("/") if it's run on Linux.

Reply:
The path as it's now written will work on _both_ Windows and Linux without the need for an if/else.

On Windows, Python will treat the forward slashes just like they are backslashes. So '//' gets interpreted as '\\'.

On Linux, Python will treat double forward slashes just like they are single backslashes, so '//' gets interpreted as '/'.

Reply:
that's good to know :) I have been suffering unnecessarily cause I didnt understand that part.

Reply:
Paths are a pain! I was pretty excited when I discovered the above. But I just have to remember to put that double forward slash at the front!

Also, there's some weirdness going on with my comment above. Regarding Windows, I meant to say So '//' gets interpreted as "double-backslash". When I actually type the double-backslash, github is rendering it as a single backslash. I think that's because a backslash is actually an escape character in most places.

And ironically, I suspect that we actually have circa-1983 Paul Allen to thank for this!
![image](https://user-images.githubusercontent.com/19944442/118694333-d4662580-b7c0-11eb-97ce-996cd8bb78e8.png)